planet/__version__.py

```diff
@@ -1 +1 @@
-__version__ = '2.8dev'
+__version__ = '2.9dev'
```

Comment:
Should be version `2.8` when you tag the release from this branch.  Then push another commit after you tag the release to update to `2.9dev` before merging to main (https://github.com/planetlabs/planet-client-python/blob/main/RELEASE.md#release-workflow).  IMO this release workflow is verbose and overcomplicated.  We'll have a simplified release workflow before the end of this half (hopefully).

Reply:
☝️ disregard, I see you already did all of this!